src/router/rockets.test.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/extensions
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Rockets from './rockets';
+import store from '../Redux/store';
+
+jest.mock('axios');
+
+it('renders Correctly', () => {
+  const tree = render(
+    <Provider store={store}>
+      <Rockets />
+    </Provider>,
+  );
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Great job with testing. Kindly add a snap also for missions. `...../missions.test.js`

Reply:
done.